ui/src/app/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -6,20 +6,25 @@ import {uiUrl} from '../../../shared/base';
 import {PhaseIcon} from '../../../shared/components/phase-icon';
 import {Timestamp} from '../../../shared/components/timestamp';
 import {formatDuration, wfDuration} from '../../../shared/duration';
-import {WorkflowLabels} from '../workflow-labels/workflow-labels';
+import {services} from '../../../shared/services';
+import {WorkflowDrawer} from '../workflow-drawer/workflow-drawer';
 
 interface WorkflowsRowProps {
     workflow: models.Workflow;
     onChange: (key: string) => void;
 }
-export class WorkflowsRow extends React.Component<WorkflowsRowProps, {hideLabels: boolean}> {
+
+export class WorkflowsRow extends React.Component<WorkflowsRowProps, {hideDrawer: boolean; workflow: models.Workflow}> {
```

Comment:
Can we make `{hideDrawer: boolean; workflow: models.Workflow}` a format type like we do with props?

```ts
interface WorkflowRowState {
...
}
```